kill_list.md

```diff
@@ -2,4 +2,5 @@
 * Functions with side effects
```

Comment:
Awesome job! Go ahead and merge this back into master.

Reply:
ok